packages/fast-form-generator-react/src/form/form-item.select.tsx

```diff
@@ -53,7 +58,17 @@ class FormItemSelect extends React.Component<IFormItemSelectProps & IManagedClas
      * Handles the onChange of the select element
      */
     private handleChange = (event: React.FormEvent<HTMLSelectElement>): void => {
-        this.props.onChange(this.props.dataLocation, (event.target as HTMLSelectElement).value);
+        switch (this.props.propertyType) {
+            case "boolean":
+                this.props.onChange(this.props.dataLocation, (event.target as HTMLSelectElement).value === "true");
+                break;
+            case "number":
+                this.props.onChange(this.props.dataLocation, parseInt((event.target as HTMLSelectElement).value, 0));
```

Comment:
I think the "number" case is accidently working - according to MDN the radix should be between 2 and 36. I think you're looking for a radix of 10.

Reply:
Not exactly. It can be octal or decimal. Do we want to be explicit just to ensure our exact scenario? The default is actually 0 or undefined.

Reply:
Yea I seem to remember this function not working unless a radix was passed, however it seems to now. I think this is moot per our convo - we should change this to not expect a new property and just use JSON to stringify and parse the values. That will align us with the JSON schema spec where an enum can be any valid JSON data.

Reply:
Agree - the update has been pushed. You probably just need to refresh :)